DenkiAo-3244/README.md

```diff
@@ -0,0 +1,54 @@
+# Hello, dear netizen! 🖖
+
+My name is David (aka *Dave*), I'm currently working as a front-end web developer in this big world. Day by day I strive to learn more and apply it to the projects in which I am present, always looking for that special connection with the end user, making their experience easier and friendlier.
+
+Let's work together! 🔥
+
+**Do you want to visit my GitHub profile? [Click here 🔗](https://github.com/dibarradev/dibarradev)**
```

Comment:
Hello Dave, I have seen as a good practice to put the link above the text to which you refer, in this case (github profile)

Reply:
Hi Psychobroder, thanks a lot for the tip, I'll keep that in mind. 👊